effeciency testing.py

```diff
@@ -0,0 +1,24 @@
+import pandas as pd
+import math
+EffMap = pd.read_csv('analysis/Eff228.csv')
```

Comment:
Would strongly recommend integration of this into the vehicle params - the motor efficiency map is a fundamental property of the vehicle, so maybe store the interpolated object in vehicle params and parse the csv there

Reply:
Refer to comment above - this file is removed in latest version